### PR TITLE
Support for checkout blocks

### PR DESCRIPTION
**Pull request by yashgit891**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: checkout-block.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (abdulwahidsharief) [inline checkout-block.php:1]: @yashgit891 remove blank line

File: checkout-block.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                'wc-settings',
+                'wp-element',
+                'wp-html-entities',
+                'wp-i18n',
+            ],
+            null,
+            true
+        );
+
+        if(function_exists( 'wp_set_script_translations' ) ) {
```
Comment (abdulwahidsharief) [inline checkout-block.php:31]: @yashgit891 flower bracket should start with new line

File: checkout-block.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                wp_set_script_translations( 'razorpay-blocks-integration');
```
Comment (abdulwahidsharief) [inline checkout-block.php:32]: @yashgit891 remove whitespace before '

**Review by ramth05 — APPROVED** [review]
LGTM

File: woo-razorpay.php [hunk 4]
```diff
@@ -47,6 +48,38 @@
...
+    if (class_exists('Automattic\WooCommerce\Blocks\Payments\Integrations\AbstractPaymentMethodType')) 
+    {
+        require_once dirname( __FILE__ ) . '/checkout-block.php';
+
+        add_action(
+          'woocommerce_blocks_payment_method_type_registration',
+          function(PaymentMethodRegistry $payment_method_registry) {
+            $container = Automattic\WooCommerce\Blocks\Package::container();
+            $container->register(
+                WC_Razorpay_Blocks::class,
```
Comment (ChetanGN) [inline woo-razorpay.php:71]: This class name space is not defined, right? Did we test it?
Should we add namespace?
`\Your\Namespace\WC_Razorpay_Blocks::class`

**Pull request merged**